install/install.md

```diff
@@ -210,9 +244,9 @@ docker run --gpus all nvcr.io/nvidia/k8s/cuda-sample:nbody nbody -gpu -benchmark
 **4b. Legacy Docker Users.** Docker CE v18 & [nvidia-docker2](https://github.com/NVIDIA/nvidia-docker/wiki/Installation-(version-2.0)){: target="_blank"} users will need to replace the following for compatibility:
 `docker run --gpus all` with `docker run --runtime=nvidia`
 
-<br/><br/>
+<br/>
```

Comment:
Question: do we need to tell users how to get into a bash prompt when using the new containers? Greg asked this when using the new containers (since they drop you into an ipython shell by default) and I think this is not obvious to a lot of folks who aren't container experts. I'm not sure what the old container default behavior was, but that'd be helpful to know if it used bash or something else when no entry point was provided. Is there a good place to include a bit of guidance like `docker run ... /bin/bash`?

Reply:
Maybe at the top in the troubleshooting 23.08 docker overhaul highlights?

Reply:
That’d be fine with me. I think that’s mostly important if the old containers behaved differently. Then you could consider putting it in the “major differences from old containers” section.

Reply:
Added in https://github.com/rapidsai/docs/pull/414/commits/898912f4996959d4f61872c3fb3e18ab97f30066

Reply:
The `Breaking changes` section of https://github.com/rapidsai/docker/issues/539 addresses many situations to get the "old" functionality back.